cli/src/command/maintenance.rs

```diff
@@ -48,10 +49,16 @@ pub enum Action {
     /// Clear consensus messages cache.
     #[structopt(name = "clear-cache")]
     ClearCache,
+    /// Restart migration script.
+    #[structopt(name = "restart-migration")]
+    RestartMigration {
+        /// Name of the service to restart.
```

Comment:
nit: is it string representation of instance identifier? An example would be helpful for users, for sure.